lib/tools/android-manifest.js

```diff
@@ -159,7 +159,10 @@ async function extractApkInfoWithApkanalyzer (localApk) {
  */
 manifestMethods.packageAndLaunchActivityFromManifest = async function (localApk) {
   const apkInfoGetters = [
-    async () => await extractApkInfoWithApkanalyzer(localApk),
+    async () => {
+      const apkanalyzerPath = await getApkanalyzerForOs(this);
```

Comment:
Any reason to do this here rather than in the `extractApkInfoWithApkanalyzer` function?

Reply:
we need `this` context

Reply:
Ah! Makes sense.